### DISCUSSION TITLE
Android job running at foreground frequency when in background

### DISCUSSION THREAD

**Discussion by jmichas** [shinyorg/shiny#702] [Q&A] [answered]
Hi, 

I have a job that is set to run as a foreground job on Android 11. 
The foreground frequency is 1 minute. 
When testing in the emulator I backgrounded the app and the job began to run (not sure the interval from backgrounding to first run). The job continued to run in the background but at a frequency of 1 minute not the PeriodicTime of 15 minutes I entered for the JobInfo.

Is this normal? I thought the max frequency on Android was 15 minutes.

Thanks.

**Comment by aritchie** [comment, marked as answer]
You likely have another service (ie. GPS) or something that is keeping the app alive.  There isn't anything I do special for foreground jobs other than starting/stopping a timer.

**Reply by jmichas** [reply]
For anyone else finding this, I was running XF5 + Prism + Unity + Shiny using Shiny.Prism to wire it all up. After switching to XF5 + Prism + Shiny.Framework + DryIoc. Everything seems to be working fine.
